### PR TITLE
meetings: several

### PR DESCRIPTION
**Pull request by joesepi**
closes: #630
closes: #632
closes: #633
closes: #636
closes: #642
closes: #643
closes: #644
closes: #645

### CODE REVIEW

**Review by WaleedAshraf — APPROVED** [review]
🔨  LGTM

File: meetings/2020-09-03.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=i8MiWJbPUNw&ab_channel=node.js
+* **GitHub Issue**: https://github.com/nodejs/community-committee/issues/642
+* **Minutes Google Doc**: https://docs.google.com/document/d/1-mpHsmKOyHpP1p-SAnzO0C_CxaJTNTqQY555uDEyokc/edit
+
+## Present
+
+* CommComm Members: @nodejs/community-committee
```
Comment (bnb) [inline meetings/2020-09-03.md:11]:
```suggestion
```

File: meetings/2020-09-17.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Present
+
+* Joe Sepi (@joesepi)
+* Divy Tolia (@designMoreWeb)
+* Rachel White (@rachelnicole)
+* A.J. Roberts (@detrohutt)
+* Alex Hughes (@lonelytree)
+* Rachel Romoff (@rromoff)
+*Ademola Sobaki (@Demola)
```
Comment (bnb) [inline meetings/2020-09-17.md:17]:
```suggestion
* Ademola Sobaki (@Demola)

```

File: meetings/2020-10-01.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## Present
+
+* Tierney Cyren (@bnb)
+* Michael Dawson (@mhdawson)
+* Joe Sepi (@joesepi)
+* David Guttman (@davidguttman)
+* Alex Hughes (@lonelytree)
+* Divy Tolia
+* Rachel White
```
Comment (bnb) [inline meetings/2020-10-01.md:17]:
```suggestion
* Rachel White (@rachelnicole)
```

File: meetings/2020-10-01.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+* **Minutes Google Doc**: https://docs.google.com/document/d/1n4yHNF9HUMRSQMRLQY2DIzpqmlBrIRb2MTPdONjguMs/edit
+
+## Present
+
+* Tierney Cyren (@bnb)
+* Michael Dawson (@mhdawson)
+* Joe Sepi (@joesepi)
+* David Guttman (@davidguttman)
+* Alex Hughes (@lonelytree)
+* Divy Tolia
```
Comment (bnb) [inline meetings/2020-10-01.md:16]:
```suggestion
* Divy Tolia (@designmoreweb)
```

File: meetings/2020-10-15.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  * Optimizing proposal process
+ 
+### Issues and PRs 
+
+*Extracted from **cc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/TSC
+
+* feat(charter): introduce emergency and async votes [#919](https://github.com/nodejs/TSC/pull/919)
+
```
Comment (bnb) [inline meetings/2020-10-15.md:72]:
```suggestion
  * Tierney to submit PR to CommComm to match the charter changes
```

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by bnb — APPROVED** [review]

**Comment by bnb** [comment]
Thank you for merging @WaleedAshraf ❤️

**Pull request merged**